README.md

```diff
@@ -46,7 +29,7 @@ To run the provisioner locally, you will need a similar set of configuration.
 Then run
 
 ```
-npm run compile && DEBUG=* node lib/main.js development
+npm run compile && DEBUG=* node lib/main.js server
```

Comment:
s/server/all/ and you can have the complete provisioner running and not just the server.  We should also do `NODE_ENV=development`